http/src/main/resources/reference.conf

```diff
@@ -284,11 +284,13 @@ image {
   jupyterImage =  "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:1.0.10"
   legacyJupyterImage = "us.gcr.io/broad-dsp-gcr-public/leonardo-jupyter:5c51ce6935da"
   proxyImage = "broadinstitute/openidc-proxy:2.3.1_2"
+  cryptoDetectorImage = "us.gcr.io/broad-dsp-gcr-public/cryptomining-detector:0.0.1"
```

Comment:
I pushed this manaully: https://console.cloud.google.com/gcr/images/broad-dsp-gcr-public/US/cryptomining-detector?project=broad-dsp-gcr-public&gcrImageListsize=30